src/schema/objects/metadata.yaml

```diff
@@ -2825,6 +2825,57 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Distance between the participant's eye and the screen. If no screen was used, use `n/a`.
+  anyOf:
+    - type: number
+      unit: cm
```

Comment:
I understand that cm is the most sensible "range" of the unit to express distances and sizes here, but because we don't actually use unit to annotate the number, IMHO it would be cleaner to use the base unit of `m` so when I encounter 1 for eg width I know it is 1 meter without looking up the standard to see the unit expected to be in the specific type of the width. 

May be someone could point me to prior discussion on this aspect of there was any?

FWIW I don't see cm used in similar implicit unit https://github.com/bids-standard/bids-specification/search?q=cm&type=

Reply:
I also think that cm is the typical unit for distance from screen (judging from what I know from my own studies and from what I generally see in papers); however that said, I don't feel strongly against making it `m` either.

Reply:
Talked with @mszinte.

Fine with using meters.